base/Dockerfile

```diff
@@ -90,6 +90,18 @@ RUN curl -LO "https://dl.k8s.io/release/v${KUBE_VERSION}/bin/linux/amd64/kubectl
     chmod +x kubectl && \
     mv kubectl /usr/local/bin
 
+# zlib packed in ubuntu is of version 1.12.11,
+# while newer versions give faster reads of gz files.
+ARG ZLIB_VERSION=1.3.1
+RUN curl -L -o "/tmp/zlib.zip" "https://github.com/madler/zlib/releases/download/v${ZLIB_VERSION}/zlib${ZLIB_VERSION//./}.zip" && \
+    unzip /tmp/zlib.zip -d /tmp
+WORKDIR /tmp/zlib-${ZLIB_VERSION}
+RUN ./configure && \
+    make install
+WORKDIR /
+RUN rm -rf /tmp/zlib*
```

Comment:
Its probably not a big deal but if we combine these steps into a single then if we can loose a lot of bytes from docker size. Currently, line#96 creates layer including the zip and unpacked files which are then removed in layer 102. 

but if we merge like following:
```
RUN curl -L -o "/tmp/zlib.zip" "https://github.com/madler/zlib/releases/download/v${ZLIB_VERSION}/zlib${ZLIB_VERSION//./}.zip" && \
    unzip /tmp/zlib.zip -d /tmp &&\
   cd /tmp/zlib-${ZLIB_VERSION} && \
  ./configure && \
   make install && \
   cd / && \
   rm -rf /tmp/zlib*
```

None of the temporary files are saved as docker layer.

Reply:
thanks, updated. I am a dockerfile rookie so was not sure i could use `cd` like this or not :)